clients/python/tracker_client/queries.py

```diff
@@ -94,8 +98,12 @@
 # Get scalar fields of all your orgs
 GET_ALL_ORGS = gql(
     """
-    query GetAllOrgs {
-        findMyOrganizations(first: 100) {
+    query GetAllOrgs($after: String) {
```

Comment:
wondering if this should be `getAllOrgs($after: String)`, to match other operation names in this file.

Reply:
[In the queries for the frontend](https://github.com/canada-ca/tracker/blob/016ced8d44053fae2b15cda6fdd2f3e9564fd434/frontend/src/graphql/queries.js) query names all begin with caps, do you think it would be better to change everything to match that?